charts/kubewarden-controller/Chart.yaml

```
@@ -23,9 +23,9 @@ maintainers:
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 2.0.2
+version: 2.0.3-rc1
```

```suggestion
version: 2.1.0-rc1
```

I think this should be a minor bump of `version`